docs/source/matlab.rst

```diff
@@ -0,0 +1,10 @@
+MATLAB doc
+**********
+
+.. mat:automodule:: matlab
+
+.. mat:autofunction:: GLMestimatesingletrial
+
+.. mat:automodule:: matlab.utilities
+
+.. mat:autofunction:: calccod
```

Comment:
This is all that is needed to tell Sphinx to read and render the help sections from specific matlab file